src/resources/Clusters/Cluster.ts

```diff
@@ -0,0 +1,27 @@
+import API from '../../APICore';
+import Resource from '../Resource';
+import {ClusterAgentModel, ClusterModel, ClusterStatusModel} from './ClusterInterfaces';
+
+export default class Cluster extends Resource {
+    static baseUrl = `/rest/organizations/${API.orgPlaceholder}/clusters`;
+
+    list() {
+        return this.api.get<ClusterStatusModel[]>(Cluster.baseUrl);
```

Comment:
The type is wrong
![image](https://user-images.githubusercontent.com/35579930/67129774-809eef00-f1cd-11e9-9fd6-cd0550dae156.png)